decisions/0015-env-triad-terms.md

```diff
@@ -0,0 +1,52 @@
+---
+status: proposed / in progress
+date: 2024-08-28
+deciders: Montana Smith, Mark Miller, Sierra Moxen
+consulted: Lee Ann McCue, Natalie Winans
+informed: 
+---
+# Establishing valid terms for MIxS Environmental Triad 
+
+## Context and Problem Statement
+
+Determining what to assert for each of the environmental triad slot (`env_broad_scale`, `env_local_scale`, and `env_medium`) is difficult.
+NMDC will provide a reporducible logic to provide users with a curated lists of valid terms for each slot.
+
+These terms will not be limiting, a user can still use string match to enter terms. This is a temporart allowance until user research and refinment of these queries is completed.
+Any term provided by a user that does **not** come from the provided list will be evaluated.
```

Comment:
Questions I have when reading this:
- evaluted by whom (e.g. the people listed on this ADR)?

Reply:
added https://github.com/microbiomedata/issues/pull/881/files#r1765318762